### PR TITLE
fix: Make response schema for CreateBulkDownload consistent between flag on/off

### PR DESCRIPTION
**Pull request by bchu1**
# Pull Request

Just return the ID (not currently using any of the fields anyways) if successful.

I changed the response name so it doesn't conflict with the NextGen type.

### CODE REVIEW

File: resolver-functions/CreateBulkDownload/CreateBulkDownload.ts [hunk 1]
```diff
@@ -124,11 +131,12 @@ export const CreateBulkDownloadResolver = async (root, args, context, info) => {
...
-  });
-  return res;
+  return (
+    await fetchFromNextGen({
+      args,
+      context,
+      serviceType: "workflows",
+      customQuery: runBulkDownload,
+    })
+  ).data.runWorkflowVersion;
```
Comment (rainandbare) [inline resolver-functions/CreateBulkDownload/CreateBulkDownload.ts:141]: It is possible that there would be an error from NextGen. Then data and runWorkflowVersion won't exist.

File: utils/httpUtils.ts [hunk 2]
```diff
@@ -67,7 +67,11 @@ export const postWithCSRF = async ({
       body: JSON.stringify(body),
     });
     checkForLogin(response?.url);
-    return await response.json();
+    const responseJson = await response.json();
+    console.log(
+      `Rails POST to ${url} response: ${JSON.stringify(responseJson)}`,
+    );
+    return responseJson;
```
Comment (rainandbare) [inline utils/httpUtils.ts:74]: helpful.

**Review by rainandbare — CHANGES_REQUESTED** [review]

**Comment by rainandbare** [comment]
This is a great approach. Are we assuming that an error would already be thrown from NextGen if it is not successful?

File: resolver-functions/CreateBulkDownload/CreateBulkDownload.ts (hunk 1 above)
Comment (bchu1) [reply resolver-functions/CreateBulkDownload/CreateBulkDownload.ts:141]: Yeah, so as of this PR https://github.com/chanzuckerberg/czid-graphql-federation-server/pull/303 we're now automatically propagating errors from NextGen through Fed's `errors`

**Comment by bchu1** [comment]
> This is a great approach. Are we assuming that an error would already be thrown from NextGen if it is not successful?

Yep, basically

**Review by rainandbare — APPROVED** [review]

**Pull request merged**